docs/examples.md

```diff
@@ -2,15 +2,37 @@
 
 <!-- TODO: improve the following -->
 
-## [Next.js 13's `app` Directory](https://beta.nextjs.org/docs/routing/fundamentals#the-app-directory)
+## Edge Runtime
```

Comment:
Originally wrote it here since this is where it was before. Then added it to the readme since it was touched on there. I guess I could put in the readme to refer to the examples for older versions of nextjs?

Reply:
sure sounds good to me 🙂